solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -1008,6 +1008,29 @@ public boolean isShutDown() {
     return isShutDown;
   }
 
+  /**
```

Comment:
really minor but I prefer that methods, especially private ones,  _follow_ its callers so that you can follow the code reading down without bouncing back up

Reply:
> really minor but I prefer that methods, especially private ones, _follow_ its callers so that you can follow the code reading down without bouncing back up

interesting. my method placement choice when refactoring is usually guided by what will result in an easier-to-review diff but before-or-after makes no difference in this case (no pun intended) and so relocated to place the private method after the caller as suggested, and also after another method that is called by the caller before this method.